src/ontology/mondo-edit.obo

```diff
@@ -212466,6 +212471,7 @@ synonym: "peroxisomal alanine-glyoxylate aminotransferase deficiency" EXACT [Orp
 synonym: "peroxisomal alanine:glyoxylate aminotransferase deficiency" RELATED [OMIM:259900]
 synonym: "PH1" EXACT ABBREVIATION [Orphanet:93598]
 synonym: "primary hyperoxaluria caused by mutation in AGXT" EXACT [MONDO:design_pattern]
+synonym: "Primary Hyperoxaluria Type 1" EXACT CLINGEN_LABEL [https://www.clinicalgenome.org/working-groups/dosage-sensitivity-curation/]
```

Comment:
capital letters should be removed from the synonym

Reply:
Fixed now. I added it initially/originally as requested by ClinGen.